packages/blockchain-wallet-v4-frontend/src/modals/Swap/CoinSelection/index.tsx

```diff
@@ -38,52 +36,40 @@ class CoinSelection extends PureComponent<Props> {
     return false
   }
 
-  checkBaseCustodial = (
-    side: SwapSideType,
-    values: InitSwapFormValuesType,
-    account: SwapAccountType
-  ) => {
-    if (
-      (side === 'COUNTER' &&
-        values?.BASE?.type === SwapBaseCounterTypes.CUSTODIAL &&
-        account.type === SwapBaseCounterTypes.ACCOUNT) ||
-      (side === 'BASE' &&
-        values?.COUNTER?.type === SwapBaseCounterTypes.ACCOUNT &&
-        account.type === SwapBaseCounterTypes.CUSTODIAL)
-    ) {
-      return true
-    }
-    return false
-  }
+  render() {
+    // @ts-ignore
```

Comment:
Is this ok?

Reply:
Hmm good spot, dont think I need this anymore